Asterix/processing_functions.py

```diff
@@ -1,28 +1,6 @@
 import numpy as np
 import scipy.optimize as opt
 
-# def butterworth(image, order, length):
```

Comment:
it was commented since forever. If usefull, there are towns of windows alrealdy coded in python https://docs.scipy.org/doc/scipy/reference/signal.windows.html
or 
https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.butter.html

Reply:
Apparently Tukey is the one everybody use I think